db/migrate/20200416212000_fix_record_types_for_lab_record_import_attachments.rb

```diff
@@ -0,0 +1,8 @@
+class FixRecordTypesForLabRecordImportAttachments < ActiveRecord::Migration[5.2]
+  def up
+    execute "UPDATE active_storage_attachments SET record_type='LabRecordImport' where record_type like '%LabRecordImport%'"
```

Comment:
I would use more precise condition. Maybe it doesn't matter in this case, but make sure this is not selecting the wrong rows.